projects/packages/backup/src/js/components/Admin.js

```diff
@@ -168,7 +168,22 @@ const Admin = () => {
 		return (
 			<Container horizontalSpacing={ 3 } horizontalGap={ 3 }>
 				<Col lg={ 6 } md={ 4 }>
-					<h2>{ __( 'Your cloud backups', 'jetpack-backup-pkg' ) }</h2>
+					<h2>{ __( 'Restore points created with every edit', 'jetpack-backup-pkg' ) }</h2>
+					<p>
+						{ __(
+							'No need to run a manual backup before you make changes to your site. ',
+							'jetpack-backup-pkg'
+						) }
+						<a
+							href={ getRedirectUrl( 'jetpack-blog-realtime-mechanics' ) }
+							target="_blank"
+							rel="noreferrer"
+						>
+							{ __( 'Learn more', 'jetpack-backup-pkg' ) }
+						</a>
+					</p>
```

Comment:
Instead of having 2 strings, and a trailing space at the end of the first string, I would recommend making this one single translatable string. It would be easier to translate. To create such strings, we can rely on `createInterpolateElement`:
https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/#createinterpolateelement

You can see it in action in other places of the repo.

Here is how you would typically do it:

```diff
diff --git a/projects/packages/backup/src/js/components/Admin.js b/projects/packages/backup/src/js/components/Admin.js
index fa7f94fc45..e9be87a388 100644
--- a/projects/packages/backup/src/js/components/Admin.js
+++ b/projects/packages/backup/src/js/components/Admin.js
@@ -8,8 +8,9 @@ import {
 	PricingCard,
 } from '@automattic/jetpack-components';
 import apiFetch from '@wordpress/api-fetch';
+import { ExternalLink } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { useState, useEffect } from '@wordpress/element';
+import { createInterpolateElement, useState, useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import useConnection from '../hooks/useConnection';
 import { STORE_ID } from '../store';
@@ -170,17 +171,21 @@ const Admin = () => {
 				<Col lg={ 6 } md={ 4 }>
 					<h2>{ __( 'Restore points created with every edit', 'jetpack-backup-pkg' ) }</h2>
 					<p>
-						{ __(
-							'No need to run a manual backup before you make changes to your site. ',
-							'jetpack-backup-pkg'
+						{ createInterpolateElement(
+							__(
+								'No need to run a manual backup before you make changes to your site. <ExternalLink>Learn more</ExternalLink>',
+								'jetpack-backup-pkg'
+							),
+							{
+								ExternalLink: (
+									<ExternalLink
+										href={ getRedirectUrl( 'jetpack-blog-realtime-mechanics' ) }
+										target="_blank"
+										rel="noopener noreferrer"
+									/>
+								),
+							}
 						) }
-						<a
-							href={ getRedirectUrl( 'jetpack-blog-realtime-mechanics' ) }
-							target="_blank"
-							rel="noreferrer"
-						>
-							{ __( 'Learn more', 'jetpack-backup-pkg' ) }
-						</a>
 					</p>
 
 					<h2>{ __( 'Where are backups stored?', 'jetpack-backup-pkg' ) }</h2>
```

You'll note that in the diff above, I also introduced `ExternalLink` for the link. We typically use that component in external links in the wp-admin dashboard, so it's clear when a link will take you to an external site. Here is how it looks:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/426388/177342603-d76d5e60-5772-4ccb-a98c-60864cf149d7.png">

With that in mind, it may be worth auditing the links in use in the Jetpack Backup plugin to ensure that they follow that guideline.

Reply:
Thanks for the review @jeherve, yes, thanks for catching that, Thank you, I'll get the links and strings fixed up. 👍

Reply:
Thanks, I think it looks good now. 👍